src/bindings/python/src/openvino/frontend/tensorflow/node_decoder.py

```diff
@@ -44,26 +46,45 @@ def tf_attr_to_ov(attr):
     return OVAny(tf_attr_to_numpy(attr))
 
 
+def try_decode(line: bytes):
+    assert isinstance(line, bytes), "Expected bytes, got object of type {}".format(type(line))
+    try:
+        return True, line.decode()
+    except ValueError:
+        return False, line
```

Comment:
agreed to remove it and do not change bytestream

Reply:
Removed.